### ISSUE TITLE
Hyperparameters

### ISSUE THREAD

**Issue by Dylan-Chiu** [NavyZeng/Diff-Unmix#7]
Dear Dr. Zeng,

Sorry to bother you, I would like to ask you about the approximate value ranges of the hyperparameters γaf, γse and αrec in the Cave dataset, because I tested that the PSNR in this dataset is only 19. thus I hope you give me a favor,  giving me more information about the value ranges of batch size, learning rate and hyperparameters.  

Thank you from the bottom of my heart.

Best regards
Qiu
